### DISCUSSION TITLE
LL returns nothing in Sveltekit app

### DISCUSSION THREAD

**Discussion by AlexanderJinoridze** [ivanhofer/typesafe-i18n#668] [Q&A] [answered]
I installed typesafe i18n in my newly created Sveltekit app but when I write {$LL.HI({name:"Svelte"})} it returns nothing, I did what was written in setup guide for svelte (not for Sveltekit tho, mb there is one and mb that's the reason idk)

**Comment by ivanhofer** [comment, marked as answer]
You need to load the locale you want to use and also set the store. Take a look here: https://github.com/ivanhofer/typesafe-i18n-demo-svelte/blob/master/src/App.svelte#L12-L17

If you use SvelteKit you should take a look at this example: https://github.com/ivanhofer/typesafe-i18n-demo-sveltekit

**Reply by AlexanderJinoridze** [reply]
Thank you for your quick response, this library is awesome
